deployment/values/tools.yaml

```diff
@@ -1,2 +1,6 @@
 image:
   tag: f90bd8f
+
+  # see https://wikitech.wikimedia.org/wiki/Wikimedia_Cloud_Services_team/EnhancementProposals/Decision_record_T330102_What_buildpacks_to_allow_and_include_for_toolforge_build_service_beta
+  builders:
+    - harbor-tools.wmflabs.org/toolforge/heroku-builder-classic:22
```

Comment:
`harbor-tools.wmflabs.org` does not seem to exist.

Reply:
It does not yet no, @dhinus is working on it.

I was thinking though, that we have harbor.toolsbeta.wmflabs.org, and would be nice to use the same pattern in both, but web proxies don't allow the '.' in there (not sure how it got there for toolsbeta).

Should we just change toolsbeta to use a web proxy and have a dash?
Or should we do whatever is done for toolsbeta? (I think it's a floating IP + dns record in openstack maybe)

Reply:
We were actually checking that with web proxies it seems to be enough for harbor, so we might want to move to that instead of having to manage ssl certs and such too, wdyt?

Reply:
this was changed to `https://tools-harbor.wmcloud.org` no?